<LogEntry><Time>12/14/2018 3:28:54 PM</Time><Message>System.Exception: Exception of type 'System.Exception' was thrown.
   at CourseSite.Controllers.HomeController.ChangeLang() in E:\lightmirror\CourseCenter\CourseSite\CourseSite\Controllers\HomeController.cs:line 39 in HomeController.cs</Message></LogEntry>
<LogEntry><Time>12/14/2018 3:54:27 PM</Time><Message>System.Exception: Exception of type 'System.Exception' was thrown.
   at CourseSite.Controllers.HomeController.ChangeLang() in E:\lightmirror\CourseCenter\CourseSite\CourseSite\Controllers\HomeController.cs:line 39 in HomeController.cs</Message></LogEntry>
<LogEntry><Time>12/14/2018 4:09:42 PM</Time><Message>System.Exception: Exception of type 'System.Exception' was thrown.
   at CourseSite.Controllers.HomeController.ChangeLang() in E:\lightmirror\CourseCenter\CourseSite\CourseSite\Controllers\HomeController.cs:line 39 in HomeController.cs</Message></LogEntry>
<LogEntry><Time>1/2/2019 12:54:00 AM</Time><Message>System.Data.Entity.Validation.DbEntityValidationException: Validation failed for one or more entities. See 'EntityValidationErrors' property for more details.
   at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at CourseSite.Controllers.CustomersController.Create(Customers customers) in E:\lightmirror\CourseCenter\CourseSite\CourseSite\Controllers\CustomersController.cs:line 71 in CustomersControllers.cs</Message></LogEntry>
<LogEntry><Time>1/12/2019 2:35:19 AM</Time><Message>System.Net.Mail.SmtpException: Failure sending mail. ---&gt; System.IO.IOException: Unable to read data from the transport connection: net_io_connectionclosed.
   at System.Net.Mail.SmtpReplyReaderFactory.ProcessRead(Byte[] buffer, Int32 offset, Int32 read, Boolean readLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLines(SmtpReplyReader caller, Boolean oneLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLine(SmtpReplyReader caller)
   at System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpTransport.GetConnection(ServicePoint servicePoint)
   at System.Net.Mail.SmtpClient.GetConnection()
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- End of inner exception stack trace ---
   at CourseSite.Common.Email.SendEmail(List`1 recevir, String subject, String body, Attachment attachment)
   at CourseSite.Controllers.CustomersController.Create(Customers customers) in CustomersControllers.cs</Message></LogEntry>
